frontend/src/app/components/ExperimentRowCard/index.scss

```diff
@@ -1,9 +1,12 @@
+@import '~photon-colors/colors.scss';
```

Comment:
Looks like [the paragraph font is a little too big here](https://github.com/mozilla/testpilot/blob/918ab136e99c35fd6720e4453e8825209d15eb6c/frontend/src/app/components/ExperimentRowCard/index.scss#L88) and the wrong color. Spec has it as basically this (tweaked for photon color var and "light" weight):
```
font-weight: 300;
font-size: 13px;
color: $grey-90;
line-height: 18px;
```

The [`<h3>` is a bit large, too](https://github.com/mozilla/testpilot/blob/918ab136e99c35fd6720e4453e8825209d15eb6c/frontend/src/app/components/ExperimentRowCard/index.scss#L77):
```
font-weight: 600;
font-size: 17px;
color: $grey-90;
line-height: 18px;
```